client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -1,22 +1,29 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	Gridicon = require( 'components/gridicon' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import Gridicon from 'components/gridicon';
```

Comment:
>Should `Gridicon` be under internal dependencies?

Yep, definitely should. Moved in rebased ce609d10c3defb00404acd49a3e0b59aa75facac.